proto/buf.yaml

```diff
@@ -15,40 +15,33 @@ lint:
     - tendermint
   ignore_only:
     ENUM_VALUE_PREFIX:
-      - tendermint
```

Comment:
Why remove this? As long as the `buf.yaml` manifest does not exclude the `tendermint.*` packages, they will be linted by `buf lint`. You'd have to select the file set with `--path` every time to avoid the lints on `tendermint` protos.

Reply:
https://github.com/cometbft/cometbft/pull/1703/commits/7dc757b89e51ce1720662936890fb8a5b9a9b0ee